src/cgame/cg_local.h

```diff
@@ -3775,10 +3778,20 @@ void CG_DrawUISelectedSoldier(void);
 void CG_UICurrentSquadSetup(void);
 void CG_CampaignBriefingSetup(void);
 
+typedef enum
+{
+    TIMEOUT,
+    WOUNDED,
+    DEAD,
+    NONE
+} fireteamMemberStatusEnum_t;
+
 // Fireteam stuff
 fireteamData_t *CG_IsOnFireteam(int clientNum);
 fireteamData_t *CG_IsOnSameFireteam(int clientNum, int clientNum2);
 fireteamData_t *CG_IsFireTeamLeader(int clientNum);
+fireteamMemberStatusEnum_t CG_FireTeamMemberStatus(clientInfo_t* ci);
+vec4_t * CG_FireTeamNameColor(fireteamMemberStatusEnum_t status);
```

Comment:
These don't really need to be here at all do they? Since you only ever use them in `cg_fireteamoverlay.c`

Reply:
Yes, was just doing it for consistency sake. Since nothing else was declared in the .c file.